### PR TITLE
199 lowercase binary and home

### PR DESCRIPTION
**Pull request by valNodeOne**

### CODE REVIEW

**Comment by lxgr-linux** [comment]
We would also have to fix all of our test scripts

File: app/app.go [hunk 1]
```diff
@@ -213,7 +213,7 @@ func init() {
 		panic(err)
 	}
 
-	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name)
+	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name+"d")
```
Comment (lxgr-linux) [inline app/app.go:216]: Why are we changing the node home to cardchain`d`?

File: docker-run.sh [hunk 2]
```diff
@@ -46,7 +46,7 @@ fi
 
 SEEDS=""
 echo "peers is:" $PEERS
-sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" $HOME/.Cardchain/config/config.toml
+sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" $HOME/.cardchain/config/config.toml
```
Comment (lxgr-linux) [inline docker-run.sh:49]: And not change it in all those paths?

**Review by lxgr-linux — CHANGES_REQUESTED** [review]

File: app/app.go (hunk 1 above)
Comment (valNodeOne) [reply app/app.go:216]: Thats currently the standard in most cosmos projects i checked

**Review by lxgr-linux — APPROVED** [review]
LGTM

**Pull request merged**